### DISCUSSION TITLE
Feature Request: Allow web::JsonBody<T> || web::Json<T> from request

### DISCUSSION THREAD

**Discussion by borngraced** [actix/actix-web#2763] [Q&A] [answered]
Your issue may already be reported!
Please search on the [Actix Web issue tracker](https://github.com/actix/actix-web/issues) before creating one.

## Expected Behavior

Hi team, thank you for your extraordinaire work in the actix echo system!.

There's a featured I really needed to use while building an API route in actix but it seems to be in nonexistence yet.

I would like to be able to pass `web::Json<T>` (where `T` can match any json params passed to a req) instead of explicitly specifying `T`.

Will this be possible or perhaps it's already in existence ...?

## Current Behavior

returns 
```rust
the trait bound `JsonBody<_>: FromRequest` is not satisfied
required because of the requirements on the impl of `FromRequest` for `(Data<tokio_postgres::Client>, HttpRequest, JsonBody<_>)
```

**Comment by robjtede** [comment, marked as answer]
Try `form: web::Json<serde_json::Value>`

**Reply by borngraced** [reply]
perfect! thanks
